### ISSUE TITLE
Update risk budget to reflect lower severity of Omicron

### ISSUE THREAD

**Issue by PeterSchiano** [microCOVID/microCOVID#1348]
The current risk budget on your site assumes that 10% of covid patients will have serious outcomes and cites this [April 2021 UK study](https://www.ons.gov.uk/peoplepopulationandcommunity/healthandsocialcare/conditionsanddiseases/bulletins/prevalenceofongoingsymptomsfollowingcoronaviruscovid19infectionintheuk/1april2021).  If you visit the study it links to 2022 updates of the information, but my layman's reading of the newer version didn't see the same clear statement of what % of covid patients suffer long covid or death. 

This [NYT article](https://www.nytimes.com/live/2022/01/11/world/omicron-covid-testing-vaccines) cites several studies on the lower severity of Omicron but again without the clean "x% will suffer long covid or death" statement.

Individual decision making needs to balance a higher tolerance of infection based on the lower expected severity.  (Tempered of course by your personal risk of infecting others who are compromised or not fully boosted.)  Hoping you can dedicate some resources to risk budget side of the tool so it doesn't increasingly skew towards being overly conservative.

**Comment by zerotrickpony** [comment]
Right, so the default risk budget is based mainly on long covid, an outcome that you will of course not find any studies about with respect to Omicron since it is too new. I'm also a layman, but having reviewed maybe a dozen papers about long covid, all of them appear to measure reported symptoms after a long time (6 weeks, 12 weeks, etc) as their endpoint. I've never seen a claim that long covid is predictable before it happens, so I don't know how you'd find a study dated prior to like idk June 2022 that makes an assertion about the risk of long covid and Omicron. We'll just have to wait.

In the meantime, the heart of the microcovid tool is the infection risk calculation, which produces an infection risk. You can always take it upon yourself to compare that number to a different target if you wish.

**Comment by symphonymarie** [comment]
[This](https://donford.substack.com/p/riskoflongcovid?utm_source=%2Fprofile%2F76170150-don-ford&utm_medium=reader2) is a very helpful overview of why "lower severity" is not a very useful assessment of risk. I've read most of the sources myself previously, so I can vouch for the legitimacy. It is very long but very thorough.
